aws/resource_aws_dynamodb_table.go

```diff
@@ -783,7 +786,7 @@ func updateDynamoDbPITR(d *schema.ResourceData, conn *dynamodb.DynamoDB) error {
 	return nil
 }
 
-func readDynamoDbTableTags(arn string, conn *dynamodb.DynamoDB) (map[string]string, error) {
+func readDynamoDbTableTags(arn string, conn *dynamodb.DynamoDB) (keyvaluetags.KeyValueTags, error) {
```

Comment:
I'm a little surprised this worked! 😄 I guess its because `keyvaluetags.KeyValueTags` currently is a type alias for `map[string]string` underneath the hood. Could we please flip this back to `map[string]string` for clarity and to prevent any future refactoring issues with the `keyvaluetags` package that might change that assumption? Thanks!

```suggestion
func readDynamoDbTableTags(arn string, conn *dynamodb.DynamoDB) (map[string]string, error) {
```